.wordlist.txt

```diff
@@ -681,10 +694,12 @@ toolchain
 toolchains
 toolset
 toolsets
+torchtune
 torchvision
 tqdm
 tracebacks
 tunable
+tunings
```

Comment:
As in "This is a tuning parameter; if you want the tunings to be chosen based on an average over multiple iterations, increase the allowed tuning duration."